.kilocode/commands/accessibility-fix-agent.md

```
@@ -0,0 +1,353 @@
+## Accessibility Sweep: [Date]
+
+### Unresolved Violations (Require Human Judgment)
+
+#### File: `[path/to/file.jsx]`
+**Lines:** [line numbers] | **WCAG Level:** [A/AA/AAA] | **Category:** [Category Name]
+```[language]
+<!-- Current Code -->
+[problematic code snippet]
+```
+
+**Issue:** [Specific problem with this code]
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)